nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all of the files coming with a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage the packages**.
+It is similar to an app store for any smartphone, or the Windows Store for example. One of the main differences between those and Nest (and most package managers), is that the upgrading must be manually commanded.
+On a smartphone, the applications are usually automatically upgraded overnight when you're charging your phone, if an update is available.
+Here, it's up to the user to upgrade their installed packages whenever they want or need to.
+
+To use Nest, the only interface available for now is the Command Line Interface (CLI).
+It might be a bit nerve racking for someone who has never seen a terminal or typed any command before, but don't worry, this guide is here to help you feel comfortable!
+
+## 1.2 Packages and dependencies
+Often, a package requires another one to work properly.
+Say, for example, someone develops a messenging application, which relies on encrypted messages to provide communication between two users.
```

Comment:
Instead of saying `someone develops` (which could lead to someone asking himself "I'm not a developer, do I need dependencies?"), maybe use `someone is using a messaging application` or anything that applies to anyone.